test/system/kubernetes/fluentBitSetup.sh

```diff
@@ -27,9 +29,12 @@ KEEP_PVC=false
 NAMESPACE=${NAMESPACE:-"default"}
 NAME=${NAME:-"pravega-fluent-bit"}
 TAR_NAME="pravega-logs-export.tar"
+SKIP_FORCE_ROTATE=${SKIP_FORCE_ROTATE:-"false"}
+ALPINE_IMAGE=${ALPINE_IMAGE:-"alpine"}
```

Comment:
Should we specify the image tag here such as `apline:latest` or just 'alpine' is fine?

Reply:
The fallback tag is `latest` so its implicitly included in a sense. I can add it to be a little more clear.